src/docs/src/documentation/content/xdocs/zookeeperTutorial.xml

```diff
@@ -361,7 +361,40 @@ from each one.</para>
 </programlisting>
  
 </section>
-<section id="sc_sourceListing"><title>Complete Source Listing</title>
+
+<section>
+<title>Complete example</title>
+<para>
+In the following section you can find a complete command line application to demonstrate the above mentioned
+recipes. Use the following command to run it.
+</para>
+<programlisting>
+export CLASSPATH=".:$ZOOKEEPER_HOME/*:$ZOOKEEPER_HOME/lib/*:$ZOOKEEPER_HOME/conf"
```

Comment:
we use different instructions for defining the classpath in the Administrators guide. It would be great to be consistent here. Also, is conf needed in the classpath?

Reply:
Sure, makes sense.
I've stolen this from the Hadoop Definitive Guide.
Will take a look at the other.